org-formation/720-client-vpn/_tasks.yaml

```diff
@@ -143,6 +143,7 @@ VpnAuthRoutes:
         AccessGroups:
           - "aws-admins"
           - "aws-synapse-admins"
+          - "aws-synapsedw-users"
```

Comment:
i think we need to make a corresponding sso group, like the synapseAdminGroup here https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/700-aws-sso/_tasks.yaml#L84-L86 

an alternative is to set this to `aws-synapseprod-developers` and put kevin in it. if that makes sense to you.